src/assets/simp-environment/environments/simp/manifests/site.pp

```diff
@@ -37,17 +36,8 @@
 #
 $hostgroup = 'default'
 
-# Include the simp_options class to ensure that defaults provided there can be found:
-# See the docs for more info: http://www.puppetmodule.info/github/simp/pupmod-simp-simp_options/master/
-include '::simp_options'
+# Include the SIMP base controller with the preferred scenario
+class { 'simp': scenario => $simp_scenario }
 
-# Add Puppet classes to the `classes` array in hiera to add them to the system.
-# For special cases where a class needs to be removed from the classes array, you
-# can use the `class_exclusions` array and it will be subtracted.
-$hiera_classes          = lookup('classes',          Array[String], 'unique', [])
```

Comment:
Should we actually get rid of this class inclusion? A lot of our docs spread out in simp-doc and READMEs assume use of a classes array in hiera.